CMakeLists.txt

```diff
@@ -14,6 +14,11 @@
 #     CMake files. It is related to ZLIB support which is planned to be removed.
 #     When the support is removed, the associated include_directories command
 #     will be removed as well as this note.
+# - MBEDTLS_TARGET_PREFIX:
```

Comment:
Nitpick: Add an empty line before and not need for a "sub-section" thus:

\#     will be removed as well as this note.
\#
\# - MBEDTLS_TARGET_PREFIX: CMake targets are designed ...